public/data/chapters/id.json

```diff
@@ -3,684 +3,798 @@
         "revelationPlace": "makkah",
         "transliteratedName": "Al-Fatihah",
         "versesCount": 7,
-        "translatedName": "Pembukaan"
+        "translatedName": "Pembukaan",
+        "slug": "pembukaan"
     },
     "2": {
         "revelationPlace": "madinah",
         "transliteratedName": "Al-Baqarah",
         "versesCount": 286,
-        "translatedName": "Sapi Betina"
+        "translatedName": "Sapi Betina",
+        "slug": "sapi-betina"
     },
     "3": {
         "revelationPlace": "madinah",
         "transliteratedName": "Ali 'Imran",
         "versesCount": 200,
-        "translatedName": "Keluarga Imran"
+        "translatedName": "Keluarga Imran",
+        "slug": "keluarga-imran"
     },
     "4": {
         "revelationPlace": "madinah",
         "transliteratedName": "An-Nisa",
         "versesCount": 176,
-        "translatedName": "Wanita"
+        "translatedName": "Wanita",
+        "slug": "wanita"
     },
     "5": {
         "revelationPlace": "madinah",
         "transliteratedName": "Al-Ma'idah",
         "versesCount": 120,
-        "translatedName": "Jamuan (Hidangan Makanan)"
+        "translatedName": "Jamuan (Hidangan Makanan)",
+        "slug": "jamuan-hidangan-makanan"
```

Comment:
no actionable item actually. 

In indonesia nobody refer 'Al'maidah' as `jamuan-hidangan-makanan` 😆. Not sure it how it will affect our SEO

Reply:
The data is fetched from BE.